inventory.yml.sample

```diff
@@ -176,7 +177,14 @@ all:
 
         http_store:
           ansible_host: 10.40.0.100
+        
+        tftp_host
+          ansible_host: 10.40.0.100
+          tftp_directory: /var/lib/tftpboot/ 
 
+        httpd_pxe_host:
```

Comment:
Do we need this? I thought we use the `http_store`

Reply:
I will delete that